ctru-rs/src/gfx.rs

```diff
@@ -31,6 +64,16 @@ pub enum Side {
     Right,
 }
 
+/// A handle to libctru's gfx module. This module is a wrapper around the GSPGPU service that
+/// provides helper functions and utilities for software rendering.
+///
+/// The service exits when this struct is dropped.
+pub struct Gfx {
+    pub top_screen: RefCell<TopScreen>,
+    pub bottom_screen: RefCell<BottomScreen>,
+    _private: (),
```

Comment:
I ran Clippy against the code and it flagged this. TIL you can use `#[non_exhaustive]` on more than just enums:
https://rust-lang.github.io/rust-clippy/master/index.html#manual_non_exhaustive
```
warning: this seems like a manual implementation of the non-exhaustive pattern
  --> ctru-rs/src/gfx.rs:71:1
   |
71 |   pub struct Gfx {
   |   ^-------------
   |   |
   |  _help: add the attribute: `#[non_exhaustive] pub struct Gfx`
   | |
72 | |     pub top_screen: RefCell<TopScreen>,
73 | |     pub bottom_screen: RefCell<BottomScreen>,
74 | |     _private: (),
75 | | }
   | |_^
   |
   = note: `#[warn(clippy::manual_non_exhaustive)]` on by default
help: remove this field
  --> ctru-rs/src/gfx.rs:74:5
   |
74 |     _private: (),
   |     ^^^^^^^^^^^^
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#manual_non_exhaustive
```